### ISSUE TITLE
UI inconsistencies bug list

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#2101]
Several small UI issues to take care of:

- [x] Inconsistent drop shadows around the object. Due to the compound <div> structures shadows overlap each other and change the perception of what gets rendered. Use `overflow: hidden` to clip the extra drop shadows from the main container, effectively rendering a single shadow.
![screen shot 2018-06-27 at 4 00 49 pm copy](https://user-images.githubusercontent.com/8484061/42004327-385af36e-7a24-11e8-9e1d-e20857800087.png)

- [x] Replace the navigation header shadow with a line; this would make it easier to see the content. With the content never scrolling under, we should simplify the UI.
![screen shot 2018-06-27 at 4 00 49 pm](https://user-images.githubusercontent.com/8484061/42004329-3c78d056-7a24-11e8-84c2-b0cecbc43eaa.png)

- [x] reduce the padding on both sides of the text input to 10px;
![screen shot 2018-06-27 at 4 00 49 pm copy2](https://user-images.githubusercontent.com/8484061/42004357-672572d2-7a24-11e8-9e2f-b0296b3e2364.png)

- [x] make sure the "Today" element doesn't go over the main navigation, increase the `z-index` appropriately.

![screen shot 2018-06-27 at 4 00 40 pm](https://user-images.githubusercontent.com/8484061/42004392-80e4e054-7a24-11e8-8990-3fe8d1a3e7d1.png)

If the `overflow: hidden` is used, make sure the instance of the discussion component is set back to 
"visible" as the "Today" component would be cut in half.

- [x] When the Today separator is visible, increase the negative top position to `-11px` to overlay the navigation bottom line and not create a double line:
![screen shot 2018-06-27 at 4 11 50 pm](https://user-images.githubusercontent.com/8484061/42004483-d2cd8e5c-7a24-11e8-87f2-ca3c181dc4e8.png)

vs.
![screen shot 2018-06-27 at 4 11 43 pm](https://user-images.githubusercontent.com/8484061/42004488-d80d181a-7a24-11e8-9016-0672ba8cc12c.png)

- [x] Use drop-shadow declarations with variable for the colors, define as 
```
box-shadow: x y blur grow rgba($tc-gray-20, 0.75);
```

- [x] Make sure all the inputs are styled consistently (border colors, paddings, etc.). Right now they are slightly different. Might be a simple recompile/cache issue as we updated the grayscale recently.

- [x] Date separator line color should be `$tc-gray-10`

- [x] Make sure all the price numbers are aligned. The numbers in the list are shifted to the left right now
![screen shot 2018-06-27 at 4 20 30 pm copy](https://user-images.githubusercontent.com/8484061/42004795-4b1584fe-7a26-11e8-8ac9-3150990be822.png)

- [x] Make sure the elements and their labels are centered in the final list
![screen shot 2018-06-27 at 4 20 30 pm](https://user-images.githubusercontent.com/8484061/42004807-5da99b5a-7a26-11e8-9b8f-b8b837db8c94.png)

- [x] Label colors are not right at the moment.

- [x] Update the font-weight at the progress bar and footer from bold to regular. Due to Sketch rendering engine variation, the actual Chrome rendering with Bold is too heavy.

Expected:
![screen shot 2018-06-27 at 4 23 14 pm](https://user-images.githubusercontent.com/8484061/42004820-766a2330-7a26-11e8-8ea2-2b5aaba5d011.png)

**Comment by maxceem** [comment]
@vic-topcoder 

> Label colors are not right at the moment.

Could you please point out an example of such label.

**Comment by maxceem** [comment]
regarding
> Make sure all the inputs are styled consistently

I've made all element consistent. The only one question about textarea multiline fields of post editor:

<img width="978" alt="textarea-size" src="https://user-images.githubusercontent.com/146016/42028118-fdd83b86-7afd-11e8-8ec5-54f8e65d6c97.png">

a. Should we make font size at 2) and 3) same like at 1) ?
b. Should we make same **minimal** height at 2) and 3) ? which height to follow?

**Comment by maxceem** [comment]
Everything is done via https://github.com/appirio-tech/connect-app/commit/15a28f38e178537452153aac3da3faf518d7cbae except of two comments above.

**Comment by vic-topcoder** [comment]
I would reword that statement - use of HEX values instead of variables in declarations

**Comment by vic-topcoder** [comment]
>a. Should we make font size at 2) and 3) same like at 1) ?

It is fine at the moment, as the settings is a component we're going to style later

>b. Should we make same minimal height at 2) and 3) ? which height to follow?

The min. height currently is fine, I would probably try to fix the text position/line-height to display the text in the center in the text field in 3

Good job on the fixes!

**Comment by vic-topcoder** [comment]
All colors in SCSS should be variables, not pure HEX, so any use of $variables should not be replaced.

**Comment by maxceem** [comment]
>  I would probably try to fix the text position/line-height to display the text in the center in the text field in 3

In 3, it's a two line textarea which grows while typing, so I think there is no need to center. Let me know if I got you wrong.

> Label colors are not right at the moment.

Could you please point out an example of such label. I'm not sure what to fix here.

**Comment by vic-topcoder** [comment]
Colors in our SCSS project - searching for `color: #` yields 40 matches across 13 files.
![screen shot 2018-06-28 at 7 26 23 pm](https://user-images.githubusercontent.com/8484061/42069929-2b45bc5a-7b09-11e8-9d32-36dea39d7bec.png)

Searching for `background: #` yields 18 matches across 5 files
![screen shot 2018-06-28 at 7 27 17 pm](https://user-images.githubusercontent.com/8484061/42069990-617bc36e-7b09-11e8-9a6e-9bd2118d826e.png)


 The goal is to fix the entire project, not a single file. All HEX values in SCSS should be replaced with our color variables. Hope that helps.

**Comment by maxceem** [comment]
Thanks for your clarifications, they do help! Will finish asap.

**Comment by vic-topcoder** [comment]
👍

**Comment by maxceem** [comment]
Not colors have variables, so I replace with the closest one:
- `#000000` -> `$tc-black`: `#151516`
- `#262628` -> `$tc-gray-90`: `#2A2A2B`
- `#F5F5F5` -> `$tc-gray-neutral-light`: `#fafafb;`
- `#A3A3AD` -> `$tc-gray-30`: `#AAAAAB;`

done via https://github.com/appirio-tech/connect-app/commit/b8d61202150b8747d071f3870096b6d4f4963d46

This task is completed.

**Comment by vic-topcoder** [comment]
Great job!

**Comment by vikasrohit** [comment]
Closing the issue.
